lib/models/builder.js

```diff
@@ -163,17 +162,29 @@ module.exports = CoreObject.extend({
    * @method trapSignals
    */
   trapSignals: function() {
-    if (!signalsTrapped) {
```

Comment:
@stefanpenner - We already add an `captureExit.onExit` callback to call `.cleanup` (which calls `_cleanupSignals`) for each builder instance, so it basically cannot be out of balance (since the final cleanup will be done at process exit when the `onExit` callbacks are ran).  Therefore, I do not think it is possible (short of someone manually getting the running builder instance and calling `offExit` in their own code) to end out unbalanced...